### PR TITLE
Update core to 0.11.0

### PR DESCRIPTION
**Pull request by Rados13**

### CODE REVIEW

File: test/membrane_flac_plugin/integration_test.exs [hunk 1]
```diff
@@ -33,7 +35,9 @@ defmodule Membrane.FLAC.Parser.IntegrationTest do
 
   defp assert_parsing_success(filename, streaming?) do
     {in_path, out_path} = prepare_files(filename)
-    assert {:ok, pid} = ParsingPipeline.make_pipeline(in_path, out_path, streaming?)
+
+    assert {:ok, _supervisor_pid, pid} =
+             ParsingPipeline.make_pipeline(in_path, out_path, streaming?)
```
[the remark this answers is not in the document]
Comment (LVala) [reply test/membrane_flac_plugin/integration_test.exs:40]: Is there some kind of need to reference the module inside of itself?

File: test/membrane_flac_plugin/integration_test.exs [hunk 2]
```diff
@@ -46,9 +50,11 @@ defmodule Membrane.FLAC.Parser.IntegrationTest do
   defp assert_parsing_failure(filename, streaming?) do
     {in_path, out_path} = prepare_files(filename)
     Process.flag(:trap_exit, true)
-    assert {:ok, pid} = ParsingPipeline.make_pipeline(in_path, out_path, streaming?)
 
-    assert_receive {:EXIT, ^pid, reason}, 3000
+    assert {:ok, supervisor_pid, _pid} =
+             ParsingPipeline.make_pipeline(in_path, out_path, streaming?)
```
Comment (LVala) [inline test/membrane_flac_plugin/integration_test.exs:55]:
```suggestion
             make_pipeline(in_path, out_path, streaming?)
```

**Review by LVala — CHANGES_REQUESTED** [review]
- Remove `type` and add `spec` to `def_options`
- Bump versions in `README.md` and `mix.ex` in order to release it

File: test/membrane_flac_plugin/integration_test.exs (hunk 1 above)
[the remark this answers is not in the document]
Comment (Rados13) [reply test/membrane_flac_plugin/integration_test.exs:40]: Here we are inside module `IntegrationTest` and module `ParsingPipeline` is above `IntegrationTest`(We have two modules inside one file).

**Review by LVala — APPROVED** [review]

**Pull request merged**